src/components/PokeList.js

```diff
@@ -85,6 +106,18 @@ const PokeList = ({
       case SortTypes.WEIGHT:
         allPokemonsFiltered = sortByWeight(allPokemonsFiltered);
         break;
+      case SortTypes.TYPE:
+        allPokemonsFiltered = filteredByType(
+          allPokemonsFiltered,
+          selectTypeOption
+        );
+        break;
+      case SortTypes.ABIL:
+        allPokemonsFiltered = filteredByAbil(
+          allPokemonsFiltered,
+          selectAbilityOption
+        );
+        break;
```

Comment:
Keep in mind that sorting is different than filtering!  With this implementation, it will only be possible to have one filter active at a time.

Reply:
Well, one sort or filter.  That might not be the behavior you're looking for.